config/locales/diaspora/en.yml

```diff
@@ -175,6 +175,7 @@ en:
     zero: "no aspects"
     one: "1 aspect"
     other: "%{count} aspects"
+    inaspects: "in {1} aspects"
```

Comment:
We currently have no javascript translations in the mobile website and I worked around that to have translated strings in the chooser.
I thought it was not scope of this PR to add the javascript translations and migrate the mobile interface to use them instead of some other workarounds...

I could not use the `shared.aspect_dropdown.toggle` because I had to replace it with javascript and need the raw string instead of one with already numbers in it.